node-scripts/yt-description.mjs

```diff
@@ -268,25 +323,23 @@ function writeDescription(video) {
     const i = +video.data.videoNumber;
     const previousVideo = videos.find((vid) => vid.data.videoNumber == i - 1);
     const nextVideo = videos.find((vid) => vid.data.videoNumber == i + 1);
+    const challengePL = 'PLRqwX-V7Uu6ZiZxtDDRCi6uhfTH4FilpH';
 
     description += '\n';
 
     if (previousVideo)
-      description += `🎥 Previous video: https://youtu.be/${previousVideo.data.videoId}?list=PLRqwX-V7Uu6ZiZxtDDRCi6uhfTH4FilpH\n`;
+      description += `🎥 Previous video: https://youtu.be/${previousVideo.data.videoId}?list=${challengePL}\n`;
 
     if (nextVideo)
-      description += `🎥 Next video: https://youtu.be/${nextVideo.data.videoId}?list=PLRqwX-V7Uu6ZiZxtDDRCi6uhfTH4FilpH\n`;
-    description +=
-      '🎥 All videos: https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZiZxtDDRCi6uhfTH4FilpH\n';
+      description += `🎥 Next video: https://youtu.be/${nextVideo.data.videoId}?list=${challengePL}\n`;
```

Comment:
```suggestion
      description += `🎥 Next: https://youtu.be/${nextVideo.data.videoId}?list=${challengePL}\n`;
```

I think we can simplify and just say next, it will be clear from the interface it's a video.